testdata/p4tc_samples_outputs/direct_counter_example.template

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/direct_counter_example numtables 1
+
+$TC p4template create action/direct_counter_example/ingress/send_nh actid 1 \
+	param port_id type dev \
+	param dmac type macaddr \
+	param smac type macaddr
+$TC p4template update action/direct_counter_example/ingress/send_nh state active
+
+$TC p4template create action/direct_counter_example/ingress/drop actid 2
+$TC p4template update action/direct_counter_example/ingress/drop state active
+
+$TC p4template create extern/root/DirectCounter extid 101 numinstances 1 tc_acl 0x1136 has_exec_method 
+
+$TC p4template create extern_inst/direct_counter_example/DirectCounter/ingress.global_counter instid 1 \
+tc_numel 2048 tbl_bindable constructor param type ptype bit32 0 \
```

Comment:
One small fix

```suggestion
tc_numel 2048 tbl_bindable constructor param type ptype bit32 1 \
```

Reply:
By default, the first item of an enum has the value 0, it would be more appropriate to start numbering from 0 than 1.

Reply:
Okk, we can do that, but we'll have to change the doc

Reply:
I will change the doc and create PR.